src/main/java/com/nguyenquyhy/discordbridge/utils/ColorUtil.java

```diff
@@ -65,4 +68,11 @@ public static TextColor getColor(Color color) {
         }
         return result;
     }
+    public static String removeColor(String text){
```

Comment:
We would need to change a bunch of the current String Templates then, or use ````Text.of() ```, either way, quite a few things would need to be updated to switch to Text